### ISSUE TITLE
Can not get Jfa-go to send Password reset links or Announcement emails.

### ISSUE THREAD

**Issue by SC8198** [hrfee/jfa-go#243]
Jfa-go will not send password reset or Announcement emails. I have a folder inside my docker container that is mapped to the directory that jellyfin generates the .json file. But no activity from jfa-go when it get's generated. No logs nothing.

I have confirmed that the emails are sending by making a invite with a test user with my email and it goes through. 

Any ideas?

**Comment by robocrax** [comment]
Can you post your config showing the mapped folder. 

Also, have you tried accessing the docker container CLI and checking if you can see the password reset JSON files in that folder?

**Comment by SC8198** [comment]
```
[password_resets]

; settings for the password reset handler.
; enable to store provided email addresses, monitor jellyfin directory for pw-resets, and send reset pins
enabled         = true
; path to the folder jellyfin puts password-reset files.
watch_directory = /jf
; send users a link to reset their password instead of a pin. must be enabled to reset ombi password at the same time as the jellyfin password.
link_reset      = false
; instead of automatically setting the user's password to the pin, allow them to set a new password through the reset link.
set_password    = false
; base url for jfa-go. you can leave this if you have one set in "invite emails". this is necessary because using a reverse proxy means the program has no way of knowing the url itself.
url_base        = https://<my-jfa-go-domain.com>
; default language for password reset success screen.
language        = en-us
; path to custom email html
email_html      = 
; path to custom email in plain text
email_text      = 
; subject of password reset emails.
subject         = Jellyfin Password Reset
```

And yes when I go into the docker container is does show the file is made.

![image](https://user-images.githubusercontent.com/104422567/213615072-a41832c5-4fef-454c-a829-49d47e69aa9d.png)

**Comment by SC8198** [comment]
Still not able to receive these emails.

**Comment by hrfee** [comment]
Sorry I missed this issue. Is your host system Linux/Mac or Windows? Unless there's anything in the jfa-go log about failing to read it, my only idea currently is an issue with detecting file changes, which has happened before on Windows.

**Comment by chandlben** [comment]
I don't use the password reset function so I can't comment on that but what I do know is that Announce emails aren't sending. When I send an invite to an email it works no problem but only the invite link. Once the user signs up any welcome emails or any manual announce emails I try to send don't work.

**Comment by Fallenbagel** [comment]
I am also having the same issue. I am on linux, running jfa-go through docker. Jfa-go has the permissions to read/write to jellyfin's directory (which is baremetal). I can send invite mails but announce and password resets do not work.

**Comment by SC8198** [comment]
Okay wow I totally forgot about this issue. Apologies.

So I currently run Jellyfin in docker on a ubuntu machine. I have the data folders mapped into the linux host at /mnt/Jellyfin.
The password reset files get made in /mnt/Jellyfin/data which is mapped to the JFA-GO docker container as /jf/.

I have checked the JFA-GO container and verified that the files do show up but JFA-GO will not send the password reset emails. I did go into the email.json file and made sure all the users were set to true but that has not fixed the issue. 

Any other ideas? 

Thanks!
